src/lib/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -38,6 +40,61 @@ public DependencyFinderEngineHelper(IDependencyFilter assemblyFilter, MetadataRe
             _currentAssemblyName = _reader.GetString(assemblyDefinition.Name);
         }
 
+        private IList<AssemblyReferenceInformation> ComputeAssemblyReferences(MetadataReader metadataReader)
+        {
+            var refs = new List<AssemblyReferenceInformation>();
+            foreach (var handle in _reader.AssemblyReferences)
+            {
+                try
+                {
+                    var entry = _reader.GetAssemblyReference(handle);
+
+                    var name = metadataReader.GetString(entry.Name);
+                    var culture = entry.Culture.IsNil ? "neutral" : metadataReader.GetString(entry.Culture);
+                    var version = entry.Version;
+                    var pkt = MetadataReaderExtensions.FormatAssemblyInfo(metadataReader, entry).PublicKeyToken;
```

Comment:
You can just do `metadataReader.FormatAssemblyInfo(entry)` instead of lines 52-55

Reply:
fixed.